### DISCUSSION TITLE
IIS Logs

### DISCUSSION THREAD

**Discussion by techn00** [Security-Onion-Solutions/securityonion#8074] [Unsupported Versions] [answered]
Hi,
How do I send the web server Apache and IIS logs to Security Onion for monitoring and detection?

**Comment by dougburks** [comment, marked as answer]
You can send host logs using your choice of one of the options listed here:
https://docs.securityonion.net/en/2.3/host.html

**Reply by techn00** [reply]
Hi Dougburks,
I have installed Winlogbeat on the IIS server, but I am struggeling to send IIS Logs to Kibana. Can you guide me with steps to setup Winlogbeat to import the logs to Elastic.

**Reply by dougburks** [reply]
If IIS is logging to the Windows event log, then you should be able to follow the steps at https://docs.securityonion.net/en/2.3/beats.html#winlogbeat and add the IIS event log to the Winlogbeat configuration.

If instead IIS is logging to a plain log file in the filesystem, then you might need to switch from Winlogbeat to Filebeat:
https://www.elastic.co/guide/en/beats/filebeat/7.17/filebeat-module-iis.html
